### ISSUE TITLE
[BUG] LethalExpansionCore breaks compatibility with LethalExpansion

### ISSUE THREAD

**Issue by Dantoresco** [TheDeadSnake/LC-Touchscreen#15]
Basically what the tittle says, the touchscreen doesn't seem to work on custom moons, at least not on those made with LethalExpansion (Most of them)

**Comment by TheDeadSnake** [comment]
Which version of the mod are you playing on ? This should've been fixed in v1.0.8 (#13) unless v47 or a LExp. update broke compatibility again.

**Comment by James-Molzer** [comment]
Having the same issue im currently using the most recent version of the mod 1.0.8, custom moons seem to break the mod

**Comment by Dantoresco** [comment]
> Which version of the mod are you playing on ? This should've been fixed in v1.0.8 (#13) unless v47 or a LExp. update broke compatibility again.

I'm using last version of your mod, last versions of LE+LECore and last game version too

**Comment by TheDeadSnake** [comment]
After a bit of testing LethalExpansionCore breaks compatibility with LethalExpansion. Now there are several options:
 1) LethalExpansionCore fixes the compatibility issue (I do not currently know where the problem is exactly but it's related to the "isInGame" bool)
 2) I try to add a fix for LethalExpansionCore in my plugin (But that may take a while as I'm currently quite busy)
 3) Or if you use both LethalExpansion and LethalExpansionCore you can just edit the config file of LethalExpansionCore and set `UseOriginalLethalExpansion` to `true` as decribed in their [README](https://github.com/LethalMods/LethalExpansionCore/blob/main/README.md).
 
 TL;DR: For a quick fix see 3) otherwise it might take a while for either plugin to update.

**Comment by Dantoresco** [comment]
Using LethalExpansion without Core is not an option sadly, so I guess I will wait until it's fixed/if it's fixed.
Thank you for taking your time to investigate the issue!

**Comment by TheDeadSnake** [comment]
You don't need to get rid of LethalExpansionCore as I wrote in my last reply LethalExpansion and LethalExpansionCore are more or less the same plugin with a different code base. Again, read the README of LethalExpansionCore:
`
[... ]It acts as a simple library and does not add any features on its own. [...] [It] is fully "compatible" with LethalExpansion, when both of the plugins are installed at the same time LethalExpansion will automatically be disabled unless the UseOriginalLethalExpansion setting is set to true in which case LethalExpansionCore will be disabled instead [...]
`
So if you have both installed you can just change the config option for it to work

**Comment by Dantoresco** [comment]
Yes, I know that, the thing is that using that option basically re-enables all LE settings and those settings are the stuff that break compability with many other things. People use LECore mainly to disable all the bloat that comes with LE but still having access to custom moons/scraps

**Comment by TheDeadSnake** [comment]
Fixed in v1.0.9.
The issue was, that LethalExpansionCore does not update the original LethalExpansion isInGame variable (Which I thought it would do)
